alembic/versions/81e4ea901c38_initial_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Initial migration
+
+Revision ID: 81e4ea901c38
+Revises: 
+Create Date: 2024-10-29 12:11:47.051432
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '81e4ea901c38'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `downgrade` function is currently empty. Add SQLAlchemy operations here to revert the changes made in the `upgrade` function during a downgrade.